lib/Minz/Request.php

```diff
@@ -35,6 +42,7 @@ public static function params() {
 	 * @param mixed $default default value, if no parameter is given
 	 * @param bool $specialchars special characters
 	 * @return mixed value of the parameter
+	 * @deprecated use typed versions instead
```

Comment:
Making the previous method deprecated.
Still used in one location: `FreshRSS_entry_Controller->readAction()`